Plugins/Public/base_plugin/Main.cpp

```diff
@@ -1114,6 +1116,14 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved)
 			delete base->second;
 		}
 
+		for (uint customSolar : customSolarList)
+		{
+			int isAlive = pub::SpaceObj::ExistsAndAlive(customSolar);
+			if(isAlive)
+				pub::SpaceObj::Destroy(customSolar, DestroyType::VANISH);
```

Comment:
Is vanish always desired? Wont sometimes a fuse be the desired effect

Reply:
Fair, adjusting.

Worst case scenario, the object will have no fuse effect and will vanish anyway.